src/pages/EventDetails/components/EventRenderer.tsx

```diff
@@ -180,12 +180,26 @@ const EventRenderer = ({ data, preview = false }: EventRendererProps) => {
             variant='outlined'>
             Meld deg av
           </VerifyDialog>
-        ) : (
-          isFuture(startDate) && (
-            <Alert severity='info' variant='outlined'>
-              Avmeldingsfristen er passert
+        ) : isFuture(sub(Date.parse(data.start_date), { hours: 2 })) ? (
```

Comment:
Bruk heller `subHours` og `parseISO` fra date-fns
```suggestion
        ) : isFuture(subHours(parseISO(data.start_date), 2)) ? (
```